EpikCord/__main__.py

```diff
@@ -21,6 +28,35 @@ async def ready():
 
 client.login()
 """
+__gitignore_template = """# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+# C extensions
+*.so
+# Distribution / packaging
+.Python
+env/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+# Our configuration files
+config.py
+"""
+__config_file_template = """
+token = [token]
+"""
 if get_parent_os().lower() == "windows":
```

Comment:
```python
IS_WINDOWS = get_parent_os().lower() == "windows"
```

Reply:
```py
IS_WINDOWS = sys.platform.startswith("win") # For me this returns win32, not sure if it'd have win64 or win86
```

Reply:
Also, surely this would raise an `UnboundLocalVariableError` or whatever its called, you need to start it off first and *then* redefine it.